CMakeLists.txt

```diff
@@ -273,7 +273,6 @@ endif()
 find_package(Armadillo 8.400.0 REQUIRED)
 
 # Include directories for the previous dependencies.
-set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} ${ARMADILLO_INCLUDE_DIRS})
```

Comment:
I assume that we need to re-add this line?

Reply:
Yes, I think it has been removed because of a bad merge